packages/core/package.json

```diff
@@ -62,6 +62,56 @@
       "import": "./dist/env.mjs",
       "edge-light": "./dist/env.edge-light.mjs",
       "require": "./dist/env.js"
+    },
+    "./readers/AssemblyAIReader": {
```

Comment:
@himself65, what happens if we add a new reader and forget to mention it here? I assume it will be added to the main bundle.

Reply:
if that's the case i think it's good to merge. Thanks @himself65 !

Reply:
> it will be added to the main bundle.

Yes